Features/RoleUI/RoleUI.feature

```diff
@@ -0,0 +1,166 @@
+    And I click on the "Roles" link
+    Then I see the Roles page
+    And I see the following roles:
+      | Name          | ID |
+      | Anonymous     | 1  |
+      | Administrator | 2  |
+      | Editor        | 3  |
+      | Partner       | 4  |
+      | Member        | 5  |
+      | Organizers    | 6  |
+    And I see an "Edit" button for each one of the roles
+    And I see a "Create" button
+
+  @javascript @common
+  Scenario: Verify the information displayed by one of the roles
+    Given I am on the Roles page
+    When I click on the "Anonymous" link
+    Then I see the following limitations fields:
+      | Module | Function | Limitation |
+    And I should see an "Edit" button
+    And I should see a "Delete" button
+    And I should see a label for the Role Assigments
+    And I should see a group with the Role Assigments
+      | User/group | Limitation |
+
+  @javascript @edge
+  Scenario: Display the information of a role with empty policies
+    Given I have an "Organizers" role
+    And "Organizers" do not have any assigned policies
+    And "Organizers" do not have any assigned Users and groups
+    When I am viewing the "Organizers" role's details
+    Then I should see that there are no policies set up for this role
+    And I should see that this role has no Role Assigments
+
+  ##
+  #
+  # Creating Roles
+  #
+  ##
+  @javascript @common
+  Scenario: Validate the existence of expected fields when creating a role
+    Given I am on the Roles page
+    When I create a new role
+    Then I should see a "Name" field
+    And I should see a "Save" button
+    And I should see a "Cancel" button
+
+  @javascript @common
+  Scenario: create a valid role
+    Given I am on the Roles page
+    And I create a new role
+    And I set "Organizers" as the "Name"
+    When I click on "Save" button
+    Then the Role is successfully published
+
+  @javascript @edge
+  Scenario: create a valid role with japanese characters
+    Given I am on the Roles page
+    And I create a new role
+    And I set "私は日本の名前で試してみる必要が" as the "Name"
+    When I click on "Save" button
+    Then the Role is successfully published
+
+  @javascript @edge
+  Scenario: creating a Role with an empty name triggers an obligation to fill the name
+    Given I am on the Roles page
+    And I create a new role
+    And I set empty as the "Name"
+    When I click on "Save" button
+    Then I see a message asking for the field "Name" to be filled
+    And the Role is not published
+
+  @javascript @edge
+  Scenario: creating a Role with an existing identifier fails validation
+    Given I have an "Organizers" role
+    And I create a new role
+    And I set "Organizers" as the "Name"
+    When I click on "Save" button
+    Then I see a message saying that the name already exists
+    And the Role is not published
+
+  @javascript @common
+  Scenario: cancel the creation of a role
+    Given I am on the Roles page
+    And I create a new role
```

Comment:
It was a way to differentiate different conceptual things: the name of a place (the "R|r"oles page) from a name of an object (the role itself). But i see this decision is raising doubts, so it's not as clear as i tough when i wrote the sentences. Being so, it might be better to level everything and use smallness in both cases.